onepassword_rules/onepassword_lut_sensitive_item_access.py

```diff
@@ -0,0 +1,35 @@
+"""
+This rule requires the use of the Lookup Table feature currently in Beta in Panther, 1Password
+logs reference items by their UUID without human-friendly titles. The instructions to create a
+lookup table to do this translation can be found at :
+
+ https://docs.runpanther.io/guides/using-lookup-tables-1password-uuids
+
+The steps detailed in that document are requrired for this rule to function as intended.
+"""
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    sensitive_item_watchlist = ["demo_item"]
```

Comment:
[consider] put in higher namespace with a comment about customizing it?

Reply:
Yeah that will match its non LUT counterpart better as well